docs/source/trainer/logging.rst

```diff
@@ -29,34 +29,39 @@ and also saves them to the file
 ``log.txt``.
 
 .. testsetup::
-    :skipif: not _WANDB_INSTALLED or not _COMETML_INSTALLED
+    :skipif: not _WANDB_INSTALLED or not _COMETML_INSTALLED or not _NEPTUNE_INSTALLED
 
     import os
+    import logging
 
+    logging.getLogger("neptune").setLevel(logging.CRITICAL)
```

Comment:
Is this needed?

Reply:
Yes, because their doctest was checking that there was no output. Unfortunately, even our offline mode produces info and warning output that was failing their CI